projects/cds/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@spartacus/cds",
+  "version": "0",
+  "description": "Spartacus - the cds",
+  "keywords": [
+    "spartacus",
+    "storefront",
+    "angular",
+    "framework"
```

Comment:
can we make it: 

- "spartacus"
- "framework"
- "storefront"
- "merchandising"
- "segmentation"
- "context-driven services" (or separate words if that's needed)
- "cds"